.gitignore

```diff
@@ -1,36 +1,33 @@
-*.gem
-*.rbc
-/.config
-/.idea
-/coverage/
-/InstalledFiles
-/node_modules
-/pkg/
-/spec/reports/
-/spec/examples.txt
-/test/tmp/
-/test/version_tmp/
-/tmp/
+# Gradle files
+.gradle/
+build/
 
-# Used by dotenv library to load environment variables.
-# .env
+# Local configuration file (sdk path, etc)
+local.properties
 
-## Documentation cache and generated files:
-/.yardoc/
-/_yardoc/
-/doc/
-/rdoc/
+# Log/OS Files
+*.log
 
-## Environment normalization:
-/.bundle/
-/vendor/bundle
-/lib/bundler/man/
+# Android Studio generated files and folders
+captures/
+.externalNativeBuild/
+.cxx/
+*.apk
+output.json
 
-# for a library or gem, you might want to ignore these files since the code is
-# intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
-# .ruby-version
-# .ruby-gemset
+# IntelliJ
+*.iml
+.idea/
+misc.xml
+deploymentTargetDropDown.xml
+render.experimental.xml
 
-# unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
-.rvmrc
+# Keystore files
+*.jks
+*.keystore
+
+# Google Services (e.g. APIs or Firebase)
+google-services.json
+
+# Jacoco
+*.exec
```

Comment:
Shall we ignore them later when we have these 🙏

Reply:
@Wadeewee what's wrong with having them beforehand? 😆

Reply:
@AVI5HEK If you're sure we will have these in the future. It's fine to leave it like this then 🙇‍♀️ I just asked you for sure because I didn't see any dependencies file of them, Especially Firebase setup 🙏

Reply:
@Wadeewee this is part of the [gitignore](https://github.com/github/gitignore/blob/main/Android.gitignore) template for Android. We can indeed remove a few things now and add more in the future. But I don't think it's important enough to sort them out right now. Just copy-paste and call it a day. 😆

Reply:
Ah, Got it @AVI5HEK. I guess you will continue to work on this again after we finish `Nimble-growth` then 🚀